eng/pipelines/official.yml

```diff
@@ -1,191 +1,115 @@
-# Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+# Pipeline: https://dev.azure.com/dnceng/internal/_build?definitionId=1298
 
-# Name: dotnet-workload-versions-official-ci
-# URL: https://dev.azure.com/dnceng/internal/_build?definitionId=1298
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/*
 
-# Produces the signed NuGet packages for workload versioning.
-
-###################################################################################################################################################################
-# PIPELINE METADATA
-###################################################################################################################################################################
-
-# Activates the pipeline after a PR is merged.
-# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/trigger?view=azure-pipelines
-trigger: none
-
-# Activates PR builds for this pipeline.
-# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#examples
 pr: none
 
 parameters:
-# When true, tries to publish
-- name: StabilizePackageVersion
-  displayName: Stabilize Package Version
+- name: stabilizePackageVersion
+  displayName: Stabilize package version
   type: boolean
   default: false
 - name: publishToFeed
   displayName: Publish to feed
   type: boolean
   default: false
-- name: feedToPublishTo
-  displayName: Feed to publish to
+- name: feedForPublishing
+  displayName: Feed for publishing
+  type: string
+  default: public/dotnet9-workloads
+- name: createVSInsertion
+  displayName: Create VS insertion
+  type: boolean
+  default: false
+- name: usePreComponentsForVSInsertion
+  displayName: Use Preview Components for VS insertion
+  type: boolean
+  default: false
+- name: vsTopicBranch
+  displayName: 'VS Topic Branch [default: temp/{team}/{target}/yyyy-MM]'
   type: string
-  default: 'public/dotnet9-workloads'
+  default: '|default|'
+- name: workloadDropNames
+  displayName: Workload drop names
+  type: object
+  default:
+  - emsdk
+  - mono
+  - android
+# These insert (pre)components & packs into VS.
+- name: primaryVsInsertionBranches
+  displayName: Primary VS insertion branches
+  type: object
+  default:
+  - rel/d17.12
```

Comment:
The preview version is 17.12, and the non-preview version is 17.13?

Reply:
I have no idea. The primary branches insert components and packs. The secondary only insert packs. If that is related to preview versions... I don't know.

Reply:
@marcpopMSFT What are the VS branches that are appropriate here?

Reply:
Shouldn't we be using the insertion client JSON configuration to find the relevant branches for insertions?

Reply:
I think 17.12 and 17.13 are currently primary. I think 18.0 will be secondary in the workloads 9 branches eventually but that branch doesn't exist yet. 17.14 is still in preview on the VS side so when I insert those workloads in, they go into main and I usually copy from the N-1 branch so we could theoretically include main in addition to 12 and 13 until .14 snaps in april

Reply:
I've updated this to use `rel/d17.12`, `rel/d17.13`, and `main` as primary. For secondary, I've set it to an empty array (no secondary branches).